src/main/java/com/ent/staySafe/entity/StaySafe.java

```diff
@@ -84,23 +84,23 @@ public void setRoomNumber(String roomNumber) {
 	}
 
 
-	public String getCheckinDate() {
-		return checkinDate;
+	public String getCheckInDate() {
+		return checkInDate;
```

Comment:
Oracle Java Method naming conventions, "Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized."
I capitalized this method name to adhere to these conventions

https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html